src/Module.ts

```diff
@@ -1072,7 +1096,15 @@ export default class Module {
 	private addImport(node: ImportDeclaration): void {
 		const source = node.source.value;
 		this.addSource(source, node);
+
 		for (const specifier of node.specifiers) {
+			if (
```

Comment:
Maybe we an avoid the repeated property access `specifier.local.name` by extracting `name` to a variable first (which we could also use below)?